native/game_backend/src/loot.rs

```diff
@@ -46,16 +46,24 @@ impl LootConfig {
         loots
             .into_iter()
             .map(|config| {
-                let effects = effects
+                let loot_effects: Vec<Effect> = effects
                     .iter()
                     .filter(|effect| config.effects.contains(&effect.name))
                     .cloned()
                     .collect();
+
+                if loot_effects.is_empty() {
```

Comment:
You're right. I actually added it while keeping this one too, since a loot without effects doesn't make sense. Same for the GiveEffect SkillMechanics!

Reply:
Now it doesn't make sense, but eventually it could

For example once we implement the fake loot box the actual implementation could be a skill so the loot won't have effect. Regardless that's for later, so for now is more than perfect to have the empty validation